cfgov/cfgov/settings/base.py

```diff
@@ -64,18 +74,14 @@
         }
     },
     {
-        'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'NAME': 'wagtail-env',
+        'BACKEND': 'django.template.backends.jinja2.Jinja2',
+        'DIRS': [Path(os.path.dirname(__file__)).ancestor(2) + '/v1/jinja2/v1'],
         'APP_DIRS': True,
```

Comment:
Yeah it was conflicting with the sheerlike.environment but I'm pushing a change up that's much cleaner. APP_DIRS is defaulted to False so the sheerlike.environment wasn't finding it.